src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
     }
 }
```

Comment:
The class `ColorSupplier` should have an access modifier. It's a best practice to explicitly define the access level for classes to avoid the default package-private access. Consider defining this class as `public` if it's meant to be accessible from outside its package.